pom.xml

```diff
@@ -209,6 +209,7 @@ flexible messaging model and an intuitive client API.</description>
     <directory-maven-plugin.version>0.3.1</directory-maven-plugin.version>
     <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
     <maven-surefire-plugin.version>3.0.0-M3</maven-surefire-plugin.version>
+    <test.additional.args></test.additional.args>
```

Comment:
Nit: maybe this variable could be moved to the top of the sections instead of being grouped with the version variables.

Reply:
I moved this to the top.
I had added this line here next to surefire version, but actually it makes more sense to have it next to other test related properties.